### PR TITLE
PAINTROID-212: Adapt format to original image

### PR DESCRIPTION
**Pull request by harsh-hash**
Ticket Solve - https://jira.catrob.at/browse/PAINTROID-212
- first changed the format option for the loaded image
- second changed the format option for the new image

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [ ] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by ThorstenBandel** [comment]
Could you change the import order for import static androidx.test.espresso.matcher.ViewMatchers.withSpinnerText; in MenuFileActivityIntegrationTest.java:63 the import needs to be down with the other androidx.test.espresso imports
You can test if this works locally with running "checkstyle" in the "other" gradle tasks on the right handside in android studio

**Comment by harsh-hash** [comment]
@ThorstenBandel sir I have fixed the warning please let me know if you need any changes.

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java [hunk 1]
```diff
@@ -428,9 +427,6 @@ public void showSaveImageInformationDialogWhenStandalone(int permissionCode, int
 		}
 
 		boolean isStandard = false;
-		if (permissionCode == MainActivityConstants.PERMISSION_EXTERNAL_STORAGE_SAVE_COPY) {
```
Comment (ThorstenBandel) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java:431]: Can you put this part back in? With this the last failing test works as well, i just tested it locally.

Comment (harsh-hash) [reply Paintroid/src/main/java/org/catrobat/paintroid/ui/MainActivityNavigator.java]: done

**Review by ThorstenBandel — APPROVED** [review]
LGTM

**Comment by harsh-hash** [comment]
@ThorstenBandel sir I have fixed all warnings except the Regexp error ( Spaces aren't allowed for indentation. Tabs only ) in this [PR](https://github.com/Catrobat/Paintroid/pull/855). Could you tell me how to fix them? As per the error says Spaces aren't allowed for indentation. Tabs only I tried but it's not fixing it.

**Comment by wslany** [comment]
Thansk!

**Pull request merged**